EXILED/Exiled.API/Features/Player.cs

```diff
@@ -3516,6 +3516,26 @@ public void SetCooldownItem(float time, ItemType itemType)
         /// <param name="projectileType">The projectile that will create the effect.</param>
         public void ExplodeEffect(ProjectileType projectileType) => Map.ExplodeEffect(Position, projectileType);
 
+        /// <summary>
+        /// Sends to the player a Fake Change Scene.
+        /// </summary>
+        /// <param name="newSceneName">The new Scene the client will load.</param>
+        public void SendFakeSceneLoading(string newSceneName)
+        {
+            SceneMessage message = new()
+            {
+                sceneName = newSceneName,
+            };
+
+            Connection.Send(message);
+        }
+
+        /// <summary>
+        /// Sends to the player a Fake Change Scene.
+        /// </summary>
+        /// <param name="newSceneName">The new Scene the client will load.</param>
+        public void SendFakeSceneLoading(ScenesType newSceneName) => SendFakeSceneLoading(newSceneName.ToString());
+
```

Comment:
if you have put all possible scene in your enum i don't see why there need to make two method ? 
also i thinks this method would be better in MirrorExtension ? (not sure of this)

Reply:
I've added 2 methods if in the future this feature never gets updated you can directly call the Name or if you want to check out something else and the "safe" method is the one who uses the Enums.

I choosed to not put them in mirror extension because they are releated to the server and player and not something that Is "Fake" but that can affect the player and server directly

Reply:
Ok i've changed the request to meet the changes required.